src/entities/Level.hpp

```diff
@@ -41,9 +41,20 @@ class Level {
 
     Level();
 
+    /**
+     * @brief position setter function
+     *
+     * @param hPosition horizontal position
+     * @param vPosition vertical position
+     */
+    void setPosition(int hPosition, int vPosition);
```

Comment:
idem here, this function is useless if another constructor that the default one can be called inside the LevelEditorController and parameters directly sent